aten/src/ATen/native/TestOps.cpp

```diff
@@ -74,5 +74,13 @@ Tensor _test_warn_in_autograd(const Tensor &self) {
   return self.clone();
 }
 
+Tensor _test_autograd_multiple_dispatch(const Tensor &self) {
```

Comment:
maybe a quick comment here about what these are for, how the real action is in derivatives.yaml, something like that 😁  (comment above the `_test_ambiguous_defaults` overloads is basically what I'm thinking of)